posthog/clickhouse/client/execute_async.py

```diff
@@ -235,26 +215,14 @@ def enqueue_process_query_task(
     query_status = QueryStatus(id=query_id, team_id=team.id, start_time=datetime.datetime.now(datetime.timezone.utc))
     manager.store_query_status(query_status)
 
+    task_signature = process_query_task.si(
+        team.id, user.id if user else None, query_id, query_json, LimitContext.QUERY_ASYNC
+    )
```

Comment:
This refresh mode is unset just like before, so I don't quite see how "calculations are started asap instead of the frontend going back and forth" is implemented in this specific PR. But I'm probably missing something. What's the logic that makes this aspect different in this PR vs. before?

Reply:
I see, from the PR description I was unsure if it was the intention for "calculations are started asap" to be unflagged here, makes sense now :+1:

Reply:
We need to unflag and think about simplifying the dashboard logic in a follow up 👍🏻